packages/test/test-service-load/src/stressTest.ts

```diff
@@ -80,14 +73,14 @@ export async function stressTest(
 			"--seed",
 			`0x${args.seed.toString(16)}`,
 		];
-		if (args.debug === true) {
+		if (args.debug) {
```

Comment:
I've always liked the explicit boolean comparisons to protect against a potential issue at runtime like someone passing the string `"false"` for `args.debug`. I know test-service-load is not part of the API so I don't feel as strongly. Just curious if you'd go for this kind of refactor in code that's "user-facing"?

Reply:
Generally yes - if I don't trust the passed values I'd be looking to validate or normalize them in readRunOptions() (or more generally, as soon as they're received) such that the "untrustworthiness" is quarantined to as minimal a part of the system as possible.